pom.xml

```diff
@@ -25,8 +25,8 @@
       <module>014-quarkus-panache-with-transactions-xa</module>
       <module>015-quarkus-micrometer</module>
       <module>017-quartz-cluster</module>
-      <module>018-quarkus-runtime-properties</module>
-      <module>019-quarkus-consul</module>
+     <!-- <module>018-quarkus-runtime-properties</module>
+      <module>019-quarkus-consul</module> -->
```

Comment:
+1 ... also noted in https://github.com/quarkus-qe/beefy-scenarios/pull/97#discussion_r579206302